bindings/pydeck/pydeck/bindings/layer.py

```diff
@@ -82,28 +83,29 @@ def __init__(self, type, data, id=None, use_binary_transport=None, **kwargs):
         self._kwargs = kwargs.copy()
         if kwargs:
             for k, v in kwargs.items():
+                if isinstance(v, Type):
+                    continue
+
                 # We assume strings and arrays of strings are identifiers
                 # ["lng", "lat"] would be converted to '[lng, lat]'
                 # TODO given that data here is usually a list of records,
                 # we could probably check that the identifier is in the row
                 # Errors on case like get_position='-', however
 
                 if isinstance(v, str) and v[0] in QUOTE_CHARS and v[0] == v[-1]:
```

Comment:
This will also fail for `"a" + b + "c"` (results in `a" + b + "c`)

Reply:
I think the question here is what is the desired behavior? Is `"a" + b + "c"` -> `a + b + c` more desired?

Reply:
I would expect `"a" + b + "c"` to be `"a" + b + "c"` to match the behavior of `"a" + "c" + b`. The type result changing so dramatically from changing the order of operators is surprising.

Reply:
Right I'd also prefer that but it's a breaking change.